examples/cd_clf_cifar10.ipynb

```diff
@@ -18,9 +18,17 @@
   },
```

Comment:
<p>Missing end of first sentence!</p><p> </p><p>Also need to change "accuracy threshold set at 0.55" to "a p-value/significance level set at 0.05" or similar.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/alibi-detect/pull/206/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>